### PR TITLE
first pass at reviewing the example records for consistency

### PR DESCRIPTION
**Pull request by amilan17**
#198

### CODE REVIEW

File: examples/ca-eccc-msc-gdc.global-discovery-catalogue.json [hunk 1]
```diff
@@ -133,6 +133,7 @@
     },
     "links": [
         {
+            "amComment":"is there a real URL now?",
```
Comment (amilan17) [inline examples/ca-eccc-msc-gdc.global-discovery-catalogue.json:136]: @tomkralidis is there a real URL to use instead?

File: examples/ca-eccc-msc.daily-climate-observations.json [hunk 2]
```diff
@@ -146,6 +146,7 @@
         {
             "rel": "items",
             "href": "mqtts://example.org",
+            "amComment":"should we add observations to the climate discpline?",
             "channel": "origin/a/wis2/ca-eccc-msc/data/core/climate/observations",
```
Comment (amilan17) [inline examples/ca-eccc-msc.daily-climate-observations.json:150]: I think the channels in our examples should refer to valid topics
@tomkralidis

File: examples/ca-eccc-msc.hydrometric-archive.json [hunk 3]
```diff
@@ -143,12 +143,14 @@
...
         {
+            "amComment1":"technically, the GetCapabilities document is XML",
             "rel": "service",
             "href": "https://geo.weather.gc.ca/geomet-climate?lang=F&service=WMS&request=GetCapabilities",
             "type": "image/png",
             "title": "HYDROMETRIC.STATIONS"
         },
         {
+            "amComment2": "update href with real URL",
             "rel": "items",
```
Comment (amilan17) [inline examples/ca-eccc-msc.hydrometric-archive.json:154]: @tomkralidis replace example.org with real URL

File: examples/ca-eccc-msc.hydrometric-realtime.json [hunk 4]
```diff
@@ -144,6 +144,7 @@
             "title": "Open Government Licence - Canada"
         },
         {
+            "amComment":"update with real MQTT link",
             "rel": "items",
```
Comment (amilan17) [inline examples/ca-eccc-msc.hydrometric-realtime.json:148]: @tomkralidis replace example.org with real MQTT link

File: examples/ca-eccc-msc.nwp-gdps.json [hunk 5]
```diff
@@ -134,12 +134,14 @@
             "title": "Open Government Licence - Canada"
         },
         {
+            "amComment": "text/xml? spell out degrees?",
             "rel": "service",
             "href": "https://geo.weather.gc.ca/geomet?lang=en&service=WMS&request=GetCapabilities&layers=GDPS.ETA_TT",
             "type": "image/png",
             "title": "Air temperature [\u00b0C]"
```
Comment (amilan17) [inline examples/ca-eccc-msc.nwp-gdps.json:141]:
```suggestion
            "title": "Air temperature [degrees]"
```

File: examples/ca-eccc-msc.nwp-gdps.json [hunk 6]
```diff
@@ -134,12 +134,14 @@
...
         },
         {
+            "amComment":"update with real MQTT link",
             "rel": "items",
             "channel": "origin/a/wis2/ca-eccc-msc/data/core/weather/nwp/global",
             "href": "mqtts://example.org",
```
Comment (amilan17) [inline examples/ca-eccc-msc.nwp-gdps.json:147]: @tomkralidis use real MQTT link

File: examples/ca-eccc-msc.surface-weather-observations-realtime.json [hunk 7]
```diff
@@ -126,6 +126,7 @@
     },
     "links": [
         {
+            "amComment":"this URL redirects to the GetCapabilities doc",
```
Comment (amilan17) [inline examples/ca-eccc-msc.surface-weather-observations-realtime.json:129]:
```suggestion
```

Comment (amilan17) [reply examples/ca-eccc-msc.surface-weather-observations-realtime.json:129]: URL works now

File: examples/cn-cma-babj.nmic.prediction-forecast.json [hunk 8]
```diff
@@ -1,5 +1,6 @@
 {
-    "id": "urn:wmo:md:cn-cma-babj:data.core.weather.prediction.forecast.shortrange.probabilistic.global",
+    "amComment": "short-range or medium-range? global or limited-area? are all of the discipline topics relevant? Is there a real link for data access?",
```
Comment (amilan17) [inline examples/cn-cma-babj.nmic.prediction-forecast.json:2]: @Amienshxq I have a couple consistency questions:
1. short-range or medium-range? 
2. global or limited-area? 
3. are all of the discipline topics relevant? 
4. Is there a real link for data access?"

File: examples/cn-cma-babj.nmic.prediction-forecast.json [hunk 9]
```diff
@@ -73,16 +74,7 @@
                         "title": "Space Weather"
                     }
                 ],
-                "scheme": "https://github.com/wmo-im/wis2-topic-hierarchy/blob/main/topic-hierarchy/earth-system-discipline/index.csv"
-            },
-            {
-                "concepts": [
```
Comment (amilan17) [inline examples/cn-cma-babj.nmic.prediction-forecast.json:79]: @Amienshxq I removed the concept of broker from this dataset record

File: examples/cn-cma-babj.nmic.surface-based-observations.json [hunk 10]
```diff
@@ -98,20 +98,20 @@
...
+            "observations"
         ]
     },
     "links": [
         {
+            "amComment":"is there a non-dev URL?",
             "rel": "data",
             "type": "text/html",
             "title": "Raw data download (bufr4)",
             "href": "https://wisdev.kishou.go.jp/data/wis2/chn/babj/data/core/weather/surface-based-observations/synop/"
```
Comment (amilan17) [inline examples/cn-cma-babj.nmic.surface-based-observations.json:110]: data is not accessible by this URL

File: examples/us-noaa-nws.gfs-10deg.json [hunk 11]
```diff
@@ -39,12 +39,14 @@
     },
     "properties": {
         "created": "2022-06-23T18:36:00Z",
-        "updated": "2022-06-23T18:36:00Z",
+        "updated": "2024-03-13T12:00:00Z",
         "wmo:dataPolicy": "core",
         "rights": "access",
```
Comment (amilan17) [inline examples/us-noaa-nws.gfs-10deg.json:44]: @solson-nws what does "access" mean?

**Comment by tomkralidis** [comment]
Superceded by #201

**Pull request closed without merging**